### ISSUE TITLE
[A11Y] [Web] Get Started Tab Template images not labeled

### ISSUE THREAD

**Issue by shengoogle** [ampproject/ampstart#419]
ENVIRONMENT:
Build number: 56.0.2924.87 beta (64-bit)
Form factor: Chromebook
Browser: Chrome

PRE-REQUISITE
chromevox is ON

STEPS TO REPRODUCE:
1. Open https://ampstart.com
2. Navigate to Get Started Tab
3. Using chromevox, Navigate Step 1: Pick a Template
4. Navigate to template images

ACTUAL RESULTS:
Images read out template one or template 2

EXPECTED RESULTS:
Template Image should be shown and properly labeled

ATTACHMENT:
https://drive.google.com/open?id=0BwUNSMCBrCB4Ul9kZlBUT0lWTlE

**Comment by torch2424** [comment]
I have a working fix for this in my fork, can have it pulled in soon :)
